frontend/libs/models/src/lib/game.model.ts

```diff
@@ -1,13 +1,16 @@
-import { Team } from './team.model';
-
-export interface Game {
-  gameMode: GameMode;
-  teamWhite: Team;
-  teamBlack: Team;
-  pointsToWin: number;
-}
+import { TeamSchema } from './team.model';
+import { z } from 'zod';
 
 export enum GameMode {
   DEFAULT = 'DEFAULT',
   RANKED = 'RANKED',
 }
+
+export const GameSchema = z.object({
+  gameMode: z.nativeEnum(GameMode),
+  teamWhite: TeamSchema.nullish(),
+  teamBlack: TeamSchema.nullish(),
+  pointsToWin: z.number(),
+});
+
+export type Game = z.infer<typeof GameSchema> & { tableId: string };
```

Comment:
Move the tableId into the schema

Reply:
This has to be added by the frontend, as the game is no (yet) aware to which table it belongs in the database